src/tasks/che.ts

```diff
@@ -84,17 +84,22 @@ export class CheTasks {
         task: () => this.kubeTasks.podStartTasks(this.keycloakSelector, this.cheNamespace)
       },
       {
-        title: 'Devfile registry pod bootstrap',
+        title: 'Devfile Registry pod bootstrap',
         enabled: ctx => ctx.isDevfileRegistryDeployed && !ctx.isDevfileRegistryReady,
         task: () => this.kubeTasks.podStartTasks(this.devfileRegistrySelector, this.cheNamespace)
       },
       {
-        title: 'Plugin registry pod bootstrap',
+        title: 'Plug-in Registry pod bootstrap',
         enabled: ctx => ctx.isPluginRegistryDeployed && !ctx.isPluginRegistryReady,
         task: () => this.kubeTasks.podStartTasks(this.pluginRegistrySelector, this.cheNamespace)
       },
       {
-        title: 'Eclipse Che pod bootstrap',
+        title: 'Eclipse Che Dashboard pod bootstrap',
+        enabled: ctx => ctx.isDashboardDeployed && !ctx.isDashboardReady,
```

Comment:
I don't see if `ctx.isDashboardReady` is set somewhere

Reply:
Is here https://github.com/che-incubator/chectl/blob/dashboard_dpl/src/tasks/che.ts#L130-L136. There was already a implementation when dashboard deployment was added to helm